packages/preview/aviation-style-checklist/0.1.0/lib.typ

```diff
@@ -0,0 +1,132 @@
+#let small_text_size = 8pt
+#let large_text_size = 14pt
+
+#let abnormals = rgb("FF9A00")
+
+#let style_state = state("style", "0")
+
+#let topic(
+  title,
+  fill_clr: rgb("FFFFFF"),
+  body
+) = {
+  box(width: 100%, stroke: 1pt, outset: 4pt, fill: fill_clr)[
+    #v(5pt)
+    #align(center)[#text(size: 20pt, upper(strong(title)))]
+    #v(5pt)
+  ]
+  columns(2)[
+    #body
+  ]
+}
+
+#let section(
+  title,
+  fill_clr: rgb("FFFFFF"),
+  body
+) = locate(loc => {
+  if (title != none and title != "") {
+    box(width: 100%, stroke: 1pt, outset: 4pt, fill: fill_clr)[
+      #v(5pt)
+      #align(center)[#upper(strong(title))]
+      #v(5pt)
+    ]
+    linebreak()
+    v(-(small_text_size))
+    if (style_state.at(loc) == 1) {
+      box(width: 1fr, stroke: 1pt, outset: 4pt, )[
+        #v(5pt)
+        #body
+        #v(5pt)
+      ]
+    } else {
+      body
+      v(-10pt)
+      ellipse(width: 30pt, height: 20pt, fill: rgb("000000"))[
+        #align(center + horizon)[#text(size: 8pt, fill: rgb("FFFFFF"), upper(strong("END")))]
+      ]
+    }
+  }
+})
+
+#let step(a, b) = {
+  if ((a != none and a != "") or (b != none and b != "")) {
+    a; " "; box(width: 1fr,repeat[.]); " "; b; linebreak()
+  }
+}
+
+#let small_caution(
```

Comment:
Public function names in Typst generally use kebab-case. If you prefer snake_case, you can also decide to keep it as is.

Reply:
Changed, no reason not to keep with the standard.